release-notes/opensearch-release-notes-2.17.1.md

```diff
@@ -0,0 +1,125 @@
+# OpenSearch and OpenSearch Dashboards 2.17.1 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 2.17.1](https://opensearch.org/versions/opensearch-2-17-1.html) includes the following bug fixes, infrastructure and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.17/release-notes/opensearch.release-notes-2.17.1.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.17/release-notes/opensearch-dashboards.release-notes-2.17.1.md).
+
+
+## BUG FIXES
+
+
+### Opensearch Anomaly Detection
+
+
+* Bump BWC Version to 2.18 and Fix Bugs ([#1311](https://github.com/opensearch-project/anomaly-detection/pull/1311))
+
+
+### Opensearch ML Commons
+
+
+* Add bedrock batch job post process function; enhance remote job status parsing (#2955)[https://github.com/opensearch-project/ml-commons/pull/2955]
+* Fix ML task index mapping (#2949)[https://github.com/opensearch-project/ml-commons/pull/2949]
+* Fix full\_response false and no output mapping exceptions (#2944)[https://github.com/opensearch-project/ml-commons/pull/2944]
+* Fix get batch task bug (#2937)[https://github.com/opensearch-project/ml-commons/pull/2937]
+* Fix field mapping, add more error handling and remove checking jobId filed in batch job response (#2933)[https://github.com/opensearch-project/ml-commons/pull/2933]
+
+
+### Opensearch Security Analytics
+
+
+* [Alerts in Correlations] Stash context for system index ([#1297](https://github.com/opensearch-project/security-analytics/pull/1297))
+* Threat intel monitor bug fixes ([#1317](https://github.com/opensearch-project/security-analytics/pull/1317))
+
+
+### Opensearch k-NN
+
+
+* Adds concurrent segment search support for mode auto [#2111](https://github.com/opensearch-project/k-NN/pull/2111)
+* Change min oversample to 1 [#2117](https://github.com/opensearch-project/k-NN/pull/2117)
+
+
+## DOCUMENTATION
+
+
+### Opensearch Alerting Dashboards Plugin
+
+
+* Added 2.17.1 release notes. ([#1109](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/1109))
+
+
+### Opensearch ML Commons
+
+
+* Fix bedrock claude3 blueprint typo (#2962)[https://github.com/opensearch-project/ml-commons/pull/2962]
+
+
+### Opensearch Notifications
+
+
+* Add 2.17.1 release notes (#[969](https://github.com/opensearch-project/notifications/pull/969))
+
+
+### Opensearch Security Analytics
+
+
+* Added 2.17.1 release notes. ([#1331](https://github.com/opensearch-project/security-analytics/pull/1331))
+
```

Comment:
We can remove all the 2.17.1 release notes update entries, and keep the fix blurprint typo one.

Reply:
Sure Peter, do you want me to remove the version increment changes too!